docs/examples/rules.kts

```diff
@@ -121,6 +121,20 @@ val ruleSet = ruleSet(ortResult, packageConfigurationProvider) {
         }
     }
 
+    packageRule("UNMAPPED_DECLARED_LICENSE") {
+        require {
+            -isExcluded()
+        }
+
+        pkg.declaredLicensesProcessed.unmapped.forEach { unmappedLicense ->
+            error(
```

Comment:
Shouldn't we stick to a warning here to have a 1:1 mapping to the previous code?

Reply:
Done.